### ISSUE TITLE
Force touch support

### ISSUE THREAD

**Issue by gritn0** [vaclavhanzl/hacktrack#2]
Hi! Is there any way to read the pressure data from the trackpad and trigger a different function with a force feedback response?

**Comment by vaclavhanzl** [comment]
Yes, there is. It is used [here in the absolute cursor positioning](https://github.com/vaclavhanzl/hacktrack/blob/main/hacktrack#L113) (line 113) by a light one-finger touch (which is switched off by default but works).

Technically, there is quite detailed information. As I remember and understand it, you have:
* four pressure sensors in trackpad corners, numbers from these are recomputed into pressure of each individual finger (this is only an approximation for many fingers of course)
* readings of capacitive sensors from the whole trackpad are reported (apart from the finger positions) as ellipsoids approximating touch areas of individual fingers - and these change with pressure, too
